### PR TITLE
fix: support DEBUG in environment

### PR DESCRIPTION
**Pull request by henryiii**
A few updates on rereading, and support for DEBUG being set in the environment.

### CODE REVIEW

**Review by ax3l — APPROVED** [review]
LGTM, thank you! :+1:

File: setup.py [hunk 1]
```diff
@@ -28,12 +28,11 @@ def __init__(self, name, sourcedir=""):
 class CMakeBuild(build_ext):
     def build_extension(self, ext):
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
-
-        # required for auto-detection of auxiliary "native" libs
```
Comment (ax3l) [inline setup.py:32]: Do you want to remove the comment? I found this a bit surprising why this is needed, so I added it.

**Comment by ax3l** [comment]
@henryiii sorry, I added a comment after approval. Can we leave that comment in? It's pretty unclear setuptools magic why this end-of-path separator is needed

File: setup.py (hunk 1 above)
Comment (henryiii) [reply setup.py:32]: I was looking at a Python 3 only version, which has the end separator automatically (due to pathlib) and has less need of the comment. Don't think I really intended to remove the comment here.

**Pull request merged**